cli/jhipster.mjs

```diff
@@ -30,53 +31,15 @@ const __dirname = dirname(__filename);
 const currentNodeVersion = process.versions.node;
 const minimumNodeVersion = packageJson.engines.node;
 
-const BUNDLED_VERSION_MESSAGE = 'Using bundled JHipster';
-const LOCAL_VERSION_MESSAGE = "Switching to JHipster installed locally in current project's node repository (node_modules)";
-
 if (!semver.satisfies(currentNodeVersion, minimumNodeVersion)) {
   logger.fatal(
     `You are running Node version ${currentNodeVersion}\nJHipster requires Node version ${minimumNodeVersion}\nPlease update your version of Node.`
   );
 }
 
-const preferLocalArg = process.argv.includes('--prefer-local');
-const preferGlobalArg = process.argv.includes('--prefer-global') || process.argv.includes('--bundled');
-
-if (preferLocalArg && preferGlobalArg) {
-  throw new Error('--prefer-local and --prefer-bundled cannot be used together');
+if (relative(__dirname, process.cwd()).startsWith('..')) {
+  logger.warn(`Since generator-jhipster v8, jhipster command will not redirect to the locally installed generator-jhipster.
```

Comment:
```suggestion
  logger.warn(`Since JHipster v8, the jhipster command will not use the locally installed generator-jhipster.
```

Make it sound nicer for the reader of the message.